.github/workflows/deploy-to-control-plane-review-app.yml

```
@@ -359,17 +309,58 @@ jobs:
             
             return deployment.data.id;
 
+  build:
+    needs: process-deployment
+    if: needs.process-deployment.outputs.do_deploy != 'false'
+    runs-on: ubuntu-latest
+    outputs:
+      image_tag: ${{ steps.build.outputs.image_tag }}
```

actionlint (1.7.4)

property "image_tag" is not defined in object type {}

(expression)